cmake/FindrocDecode.cmake

```diff
@@ -2,31 +2,43 @@
 # 
 # MIT License
 # 
-# Copyright (c) 2025 Advanced Micro Devices, Inc.
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
```

Comment:
why is this changed back to 2024

Reply:
We are including this new file now in 2025